teams/templates/change_teams.html

```diff
@@ -0,0 +1,60 @@
+{% extends "base.html" %}
+{% load static %}
+{% load teams_tags %}
+{% block content %}
+
+    {% include 'includes/menu.html' %}
+    {% if not teams and not leftover_participants %}
+        <p>Not enough participants have registered yet.</p>
+        
+        <p>Current number of registrations: {{num_participants}}</p>
+    {% endif %}
+    <div id="team-list">
+    {% if teams %}
+    {% for team_name, team_members in teams.items %}
+    <section class="card shadow mb-3 mt-3">
+        <!-- <div class="card-header">{{team_name}}  <span style="float:right">Team Score: {{ team_members|calculate_team_level }}</span></div> -->
+        <div class="card-body">
+            <h5 class="p-orange card-title">{{team_name|format_team_name}} <span style="float:right">Team Score: <span id="{{team_name}}_score">{{ team_members|calculate_team_level }}</span></span></h5>
+            <ol class="team team-drop-area" id="{{team_name}}" ondrop="drop(event)" ondragover="allowDrop(event)">
+                {% for member in team_members %}
+                <li id="{{team_name}}-{{ forloop.counter0 }}"
+                    data-level={{member.level}}
+                    data-userid={{member.userid}}
+                    draggable="true" ondragstart="drag(event)">{{member.name}} (level: {{member.level}})</li>
+                {% endfor %}
+            </ol>
+        </div>
+    </section>
+    {% endfor%}
```

Comment:
space/formatting missing

Reply:
Not sure what I am missing here?

Reply:
Not a big issue....
Have:
`{% endfor%}`
Should be:
`{% endfor %}`

Reply:
Oh you are completely right!! Did not realize that at all haha. Thanks, @TravelTimN!!

Reply:
This is corrected now.